### PR TITLE
feat: new geocoder site

### PR DESCRIPTION
**Pull request by atymic**
New site for geocoder :D

Todo:

- [ ] Add Travis/Packagist badges to main page
- [ ] Documentation page about framework support/packages?
- [ ] Favicon?
- [ ] Extend content to fill more of the existing whitespace on the docs pages (currently 1/5 is unused)
- [ ] Switch `geocoder-php.org` domain to point to netlify

See the issue in the main repo for details:
https://github.com/geocoder-php/Geocoder/issues/1003

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) <Matt Stauffer, Anthony Terrell, Keith Damiani>
```
Comment (atymic) [inline LICENSE.txt]: Not sure if we should replace this with a different licence?

**Comment by jbelien** [comment]
Wouldn't it be better to start form an orphan branch ?

**Comment by atymic** [comment]
> Wouldn't it be better to start form an orphan branch ?

Tried that and it broke github 😢 
Will create a new PR and see if that works

**Comment by atymic** [comment]
The answer is no, you can't PR an orphan branch by the look of it:

> There isn’t anything to compare.
geocoder-php:master and atymic:new-site-orphan are entirely different commit histories.

**Comment by jbelien** [comment]
Too bad :(

**Pull request closed without merging**